_docs/protocols/2017-01-06-binary.md

```diff
@@ -222,9 +222,9 @@ Example:
 // TODO:
 ~~~
 ghci> hexEncode $ messageName (Proxy :: Proxy SendProxySK)
-
+"0102"
 ghci> hexEncode $ messageName (Proxy :: Proxy (DataMsg GtMsgContents))
-
+"020a03"
```

Comment:
It's `DataMsg GtMsgContents` who encoded here, not just `GtMsgContents`. Do you agree?